internal/resources/package/spec/spec.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package spec
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+var (
```

Comment:
Have we added the schema validations as per the generated API models?

I see very minimal validations. Please verify and update.